sdk/keyvault/azkeys/client.go

```diff
@@ -254,9 +254,9 @@ func createOCTKeyResponseFromGenerated(i generated.KeyVaultClientCreateKeyRespon
 	}
 }
 
-// CreateKey - The create key operation can be used to create a new octet sequence (symmetric) key in Azure Key Vault.
-// If the named key already exists, Azure Key Vault creates
-// a new version of the key. It requires the keys/create permission.
+// CreateOCTKey can be used to create a new octet sequence (symmetric) key in Azure Key Vault.
```

Comment:
Just wondering if this function should be CreateOctKey instead to align with the official naming?  And if there is only an oct-HSM (no oct), should the default options set HardwareProtected to true? https://docs.microsoft.com/en-us/azure/key-vault/keys/about-keys

Also, should the doc comment include info that the oct-HSM key requires a managed HSM rather than a vault?
Similarly, should we include doc comments for the other key types to indicate vault compatibility or would the service provide a useful error message in those cases where the vault or managed HSM does not support the key type?

Reply:
Good catch, Changed everything from OCT -> Oct

Reply:
To follow up on the vault compatibility - is that something we should specify in the doc comments or do we rely on the service to provide an error? For some of the other functions we do seem to specify compatibility. For example GetDeletedKey mentions that it only works for soft-delete enabled vaults.

Reply:
oct and oct-HSM are treated the same by the service. If the user is targeting a MHSM and they send a request to create a `oct` key, the service will create an `oct-HSM` key. That is the reason we have both types here.